content/operate/oss_and_stack/install/install-redis-insight/_index.md

```diff
@@ -3,7 +3,7 @@ categories:
 - docs
 - operate
 - redisinsight
-description: 'Install RedisInsite on AWS, Docker, and Kubernetes
+description: 'Install Redis Insite on AWS, Docker, and Kubernetes
 
   '
 linkTitle: Install Redis Insight
```

Comment:
should we now add aliases for the old location? I just thought of this

Reply:
should we also add aliases for the files below that are moving?

Reply:
Ugh... I don't know.

Reply:
I'm going to add them in a following commit.